recipes/qemu-user-execve/patches/0001-intercept-qexecve.patch

```diff
@@ -0,0 +1,42 @@
+This patch is extracted from Joel Holdsworth's patch submission.
```

Comment:
Can you please extract the patches in the `git format-patch` form? You should be able to download them in mbox format and `git am` them.

Reply:
This is new to me, I would need to research this a bit - The patches were not merged (or even PR??) but just submitted for discussion, I found them in a mail discussion

Reply:
Would using proot a preferable solution to the problem I was trying to resolve?

Reply:
I am working on a qemu-system recipe (no patch) in order to try it as a qemu-user for OSX (I think Wolf V. needs it). It has overlapping artifact that I try to make cleanly separated and complementary, if you have the inkling a feedback would be appreciated (https://github.com/conda-forge/staged-recipes/pull/28603). It builds on linux/osx (it did on windows, but since getting a windows OS in VM is a bit above my understanding, I skip it for now)